src/Tribe/Main.php

```diff
@@ -687,6 +690,10 @@ protected function addHooks() {
 			add_filter( 'tribe_meta_chunker_post_types', array( $this, 'filter_meta_chunker_post_types' ) );
 			tribe( 'chunker' );
 
+			// Purge old events
+			add_action( 'update_option_' . Tribe__Main::OPTIONNAME, array( tribe( 'tec.event-cleaner' ), 'move_old_events_to_trash' ), 10, 2 );
```

Comment:
instead of passing 
```
array( tribe( 'tec.event-cleaner' ), 'move_old_events_to_trash' )
``` 
we can use 
```
tribe_callback( 'tec.event-cleaner', 'move_old_events_to_trash' )
```